samples/extensions/README.adoc

```diff
@@ -302,3 +303,12 @@ Demonstrate the use of the host image extension to directly copy from a host buf
 *Extensions:* https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_extended_dynamic_state3.html[`VK_EXT_line_rasterization`], https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_extended_dynamic_state3.html[`VK_EXT_extended_dynamic_state3`]
 
 Demonstrate how to use dynamic multisample rasterization (MSAA)
+
+=== xref:./{extension_samplespath}graph_pipelines/README.adoc[Graph Pipelines]
```

Comment:
This also needs to be added to the navigation in https://github.com/KhronosGroup/Vulkan-Samples/blob/main/antora/modules/ROOT/nav.adoc